docs/components/plugins/ConfigurationDiff.tsx

```diff
@@ -34,7 +34,7 @@ export default function ConfigurationDiff({ source }) {
     return null;
   }
 
-  const renderFile = ({ oldRevision, newRevision, type, hunks, oldPath, newPath }) => (
+  const renderFile = ({ oldRevision, newRevision, type, hunks, newPath }) => (
```

Comment:
```suggestion
  const renderFile = ({ oldRevision, newRevision, type, hunks, newPath }: SomeType) => (
```

Reply:
These are types coming from `react-diff-view` library which is not typed at all 😞  But I tried to deduce some of them basing on its docs. Btw. the code is a direct copy-paste of their example.